mxcube3/ui/actions/general.js

```diff
@@ -100,6 +108,14 @@ export function getInitialStatus() {
         'Content-type': 'application/json'
       }
     });
+    const beamlineLimits = fetch('mxcube/api/v0.1/beamline', {
```

Comment:
ok. Just one issue: the diff motor info is done through two different calls for the position and the limits, whereas the beamline give position and limits with a single call. So, later in the js code there is a motorLimits var which contains diff motors and beamline limits plus beamline positions.